server/methods/catalog.app-test.js

```diff
@@ -873,7 +873,7 @@ describe("core product methods", function () {
       expect(() => Meteor.call(
         "products/updateProductPosition",
         product._id, position, tag.slug
-      )).to.not.throw(Meteor.Error, /Access Denied/);
+      )).to.not.throw();
```

Comment:
I'm not sure this is an improvement. Also it seems like the expectation description is incorrect?